channel.go

```diff
@@ -1,14 +1,33 @@
 package thingfulx
 
-import (
-	"time"
-)
+import "time"
 
 // Channel represents an individual data channel being published by a Thing out
 // there in the world somewhere.
 type Channel struct {
-	ID         string    `json:"id"`
-	Value      string    `json:"value"`
+	ID           string            `json:"id"`
+	Metadata     []ChannelMetadata `json:"metadata"`
+	Observations []Observation     `json:"observations"`
+}
+
+// ChannelMetadata is an object which contains any metadata required to
+// semantically descibe a measurement collected by a Thing.
+type ChannelMetadata struct {
+	Prop string `json:"property"`
```

Comment:
Yeah - I think that works - so if Namespace is "" then it is a raw value e.g a url or a number?